.travis.yml

```diff
@@ -172,4 +152,4 @@ deploy:
     branch: master
   local-dir: "$TRAVIS_BUILD_DIR/build/artefacts_upload"
   upload-dir: "$TRAVIS_REPO_SLUG/$TRAVIS_BUILD_NUMBER/$TRAVIS_JOB_NUMBER"
-  acl: bucket_owner_full_control
+  acl: bucket_owner_full_control
```

Comment:
Please add a newline to the end of the file.

Reply:
Done!